src/component/addeditmembers/index.tsx

```diff
@@ -172,6 +174,7 @@ function AddEditMembers(props: AddEditMembersProps) {
 
             return (
               <MemberItem
+                key={member.user.id}
```

Comment:
Arrays of children should always have a key. Fixes a bug where the "remove member" dropdown would remain open when a member was deleted.